docs/es/week12/12-2.md

```diff
@@ -0,0 +1,323 @@
+- Entrena aún con más texto
+
+Mejoras aún mayores sobre el rendimiento de BERT: el rendimiento de las tareas de respuesta a preguntas es ahora sobrehumano.
+
+
+## [Entrenamiento previo para PLN] (https://www.youtube.com/watch?v=6D4EWKJgNn0&t=4963s)
+
+Echemos un vistazo rápido a los diferentes enfoques de formación previa autodirigidos que se han investigado para el PLN.
+
+- XLNet:
+
+  En lugar de predecir todos los tokens enmascarados de forma condicionalmente independiente, XLNet predice los tokens enmascarados de forma autorregresiva en orden aleatorio
+
+- SpanBERT
+
+   Enmascara tramos (secuencias de palabras consecutivas) en lugar de tokens
+
+- ELECTRA:
+
+  En lugar de enmascarar palabras, sustituimos tokens por otros similares. Luego, resolvemos un problema de clasificación binaria tratando de predecir si los tokens han sido sustituidos o no.
+
+- ALBERT:
+
+  Un Bert liviano: modificamos BERT y lo hacemos más liviano atando los pesos a través de las capas. Esto reduce los parámetros del modelo y los cálculos involucrados. Curiosamente, los autores de ALBERT no tuvieron que comprometer mucho la precisión.
+
+- XLM:
+
+  BERT multilingüe: en lugar de introducir texto en inglés, lo hacemos en varios idiomas. Como era de esperar, aprendió mejor las conexiones entre idiomas.
+
+Las conclusiones clave de los diferentes modelos mencionados anteriormente son
+
+- ¡Muchos objetivos diferentes de preentrenamiento funcionan bien!
+
+- Crucial para modelar interacciones profundas y bidireccionales entre palabras
+
+- Grandes beneficios de agrandar el preentrenamiento, aún sin límites claros
+
+
+La mayoría de los modelos discutidos anteriormente están diseñados para resolver el problema de clasificación de texto. Sin embargo, para resolver el problema de la generación de texto, donde generamos resultados secuencialmente como el modelo `seq2seq`, necesitamos un enfoque ligeramente diferente al preentrenamiento.
+
+
+#### Entrenamiento previo para generación condicional: BART y T5
+
+BART: preentrenamiento de modelos `seq2seq` mediante la eliminación de ruido en el texto
+
+En BART, para el entrenamiento previo, tomamos una oración y la corrompemos enmascarando tokens al azar. En lugar de predecir los tokens de enmascaramiento (como en el objetivo de BERT), alimentamos la secuencia corrupta completa e intentamos predecir la secuencia correcta completa.
+
+Este enfoque de preentrenamiento `seq2seq` nos da flexibilidad para diseñar nuestros esquemas de corrupción. Podemos barajar las oraciones, eliminar frases, introducir nuevas frases, etc.
+
+BART pudo igualar a RoBERTa en las tareas SQUAD y GLUE. Sin embargo, fue el nuevo estado del arte sobre resumenes, diálogos y conjuntos de datos de control de calidad abstractos. Estos resultados refuerzan nuestra motivación por BART, siendo mejores en tareas de generación de texto que BERT/RoBERTa.
+
+### Algunas preguntas abiertas en PLN
+
+- ¿Cómo deberíamos integrar el conocimiento mundial
```

Comment:
Does the translation "conocimiento del mundo real" sound more natural to you? Ditto elsewhere in the document.

Reply:
What do you think about "conocimiento global"?